src/main/java/com/example/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,24 @@
 package com.example.onlinebookstore.dto;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @Size(min = 1, max = 100)
     private String title;
+    @NotNull
+    @Size(min = 1, max = 40)
     private String author;
+    @NotNull
+    @Size(min = 10, max = 13)
```

Comment:
Think about using org.hibernate.validator.constraints.ISBN annotation for validation.